### PR TITLE
Added Swedish

### PR DESCRIPTION
**Pull request by swedentuxsteve**
Put my name as "tux_steve" or "swe_tux_steve" in the credits.txt file

# SCP:SL Translation PR Template

## Language Name
Swedish

## State Type of change (New language, changed existing Etc.)
New language

# Checklist:
- [x] I have tested the translation changes ingame
- [X] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [X] I have made sure that I'm **not** submitting a duplicate translation 
- [X] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation

### CODE REVIEW

**Comment by Dat-Joseph** [comment]
Please be sure to include all latest patches we made to our repo. There are still some missing files like abilities.txt, keybinds.txt etc...

File: Translations/sv/Version_7-0-0.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DU DOG
+Tryck på {KEYBINDING_INTERACTION} föt att placera <color=yellow>Weapon Manager-surfplatta</color>.
+Tryck på {KEYBINDING_INTERACTION} för att plocka upp <color=yellow>Weapon Manager-platta</color>.
```
Comment (WinterBeyond) [inline Translations/sv/Version_7-0-0.txt:3]: Använd antingen `Weapon Manager-surfplatta` eller behåll `Weapon Manager Tablet`. Alternativ använd `Vapenhanterare-surfplatta` för en mer hel översättning.

File: Translations/sv/Summary.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+omgången är klar!
+SCPs VANN
+MTF VANN
+KLASS-D VANN
+INGEN VANN
+De ansvarar för har dödat [summary_scpfrags] personer.
```
Comment (WinterBeyond) [inline Translations/sv/Summary.txt:6]: Ändra `för har dödat` till `för att ha dödat`.

File: Translations/sv/Summary.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+omgången är klar!
+SCPs VANN
+MTF VANN
+KLASS-D VANN
+INGEN VANN
+De ansvarar för har dödat [summary_scpfrags] personer.
+Hoten dog. Den här platsen är nu säker igen.
+All Stiftelse-säkerhet har avslutats.
```
Comment (WinterBeyond) [inline Translations/sv/Summary.txt:8]: `All säkerhetspersonal har utplånats.`

File: Translations/sv/SCP079.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+AUXILIAKRAFT
```
Comment (WinterBeyond) [inline Translations/sv/SCP079.txt:1]: `AUXILIÄRKRAFT`

File: Translations/sv/SCP079.txt [hunk 5]
```diff
@@ -0,0 +1,17 @@
+AUXILIAKRAFT
+ERFARENHET
+KAMERAFLÖDE ONLINE
+KLICKA PÅ [KEY] ATT VÄXLA KAMERA I ÖVERVAKNINGSKARTAN
```
Comment (WinterBeyond) [inline Translations/sv/SCP079.txt:4]: Ändra `ATT VÄXLA KAMERA I ÖVERVAKNINGSKARTAN` till `FÖR ATT VISA ÖVERVAKNINGSKARTAN`

File: Translations/sv/NewMainMenu.txt [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+Tillbaka
+Denna ändring kommer att kräva en omladdning.\nLadda om menyn nu?
+Ja
+Nej
+Är du säker på att du vill avsluta?
+Skärmupplösning
+Göm IP
+Välkommen tillbaka, {username}!
+Medverkande
+AUTENTISERANDE
```
Comment (WinterBeyond) [inline Translations/sv/NewMainMenu.txt:68]: `AUTENTISERAR`

File: Translations/sv/NewMainMenu.txt [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+Ansluter till centrala servrar...
+<color=green>Kontakt etablerad</color>
```
Comment (WinterBeyond) [inline Translations/sv/NewMainMenu.txt:70]: Ändra `kontakt` till `anslutning`

File: Translations/sv/NewMainMenu.txt [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+Lämna
+Social Media
+Internet
+Favoriter
+Officiella
+Historik
+Vänner
+Sök Servrar
+Skapa Server
+Joina om
```
Comment (WinterBeyond) [inline Translations/sv/NewMainMenu.txt:13]: `Återanslut`

File: Translations/sv/NewInventory.txt [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+Granat:Boom!
+Flashbang Granat:Bländar dina fiender.
+Avväpnare:Vänsterklicka avväpnar, högerklickar släpper loss.\nHåll E på någon som är avväpnad i ditt team för att befria dem.
+Släppt 7.62:No description.
+Släppt 9mm:No description.
+USP:No description.
+SCP-018:Superboll kan studsa med extrem effektivitet.
+SCP-268:När du tar på dig den då gör den dig osynlig temporärt.
+Adrenaline:Ökar tillfälligt uthållighet.
+Painkillers:Relieve minor injuries.
```
Comment (WinterBeyond) [inline Translations/sv/NewInventory.txt:35]: `Åtgärda mindre skador.`

File: Translations/sv/NewInventory.txt [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+SCP-207:Ökar snabbheten skadligt.
+Vapen Hanterar-Platta:Används för att släppa ammunition och uppgradera dina vapen.
+MTF-E11-SR:No description
+Project 90:No description
+Släppt 5.56:No description
+MP7:No description
+Logicer:No description
+Granat:Boom!
+Flashbang Granat:Bländar dina fiender.
+Avväpnare:Vänsterklicka avväpnar, högerklickar släpper loss.\nHåll E på någon som är avväpnad i ditt team för att befria dem.
```
Comment (WinterBeyond) [inline Translations/sv/NewInventory.txt:28]: Ändra `team` till `lag`

File: Translations/sv/NewInventory.txt [hunk 11]
```diff
@@ -0,0 +1,36 @@
+Vaktmästar Kort:No description
+Forkars Kort:No description
+Forskningshandledare Kort:No description
+Zonchefs Kort:No description
+Vakt Kort:No description
+MTF Kadett Kort:No description
+Inneslutnings ingenjör Kort:No description
+MTF Löjtnant Kort:No description
+MTF Kommandör Kort:No description
```
Comment (WinterBeyond) [inline Translations/sv/NewInventory.txt:9]: `Kommendör`

File: Translations/sv/MainMenu.txt [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+Som standard kommer servern att konfigureras i det lokala nätverket.\nDina vänner behöver den direkta IP-adressen för att ansluta.\nOm du vill ställa in en offentlig server väljer du LocalAdmin.exe och följer instruktionerna.\nOm du har några frågor, fråga vår tekniska support (länk i huvudmenyn)
+1. Grunderna
+Det presenterade spelet finns fortfarande i en offentlig beta.\nDu kan få problem med anslutningen och många andra buggar.\nDet aktuella tillståndet återspeglar inte den slutliga versionen.
+4. (mer handledning senare)
+DIREKTKOPPLING
+TEKNISK SUPPORT / FORUM
+LJUDVOLYM
+Ange IP-adressen för den specifika servern.
+NEDLADNING AV DATA
+INGEN SERVRAR TILLGÄNGLIGA
```
Comment (WinterBeyond) [inline Translations/sv/MainMenu.txt:55]: Ändra `INGEN` till `INGA`

**Review by OneKoin — CHANGES_REQUESTED** [review]
You should add ones that are updated too

**Comment by Rnen** [comment]
image
Please attend these issues

**Comment by Dat-Joseph** [comment]
Hello swedentuxsteve,
you did not reply to us in the past month to the earlier mentioned issues. 
Could you please resolve them within a week? If you cannot do that, we are going to close this pullrequest.
We recommend to resolve the mentioned issues before re-opening this pull request.
Thank you.

**Comment by Rnen** [comment]
No response, closing.

**Pull request closed without merging**